webapp/channels/src/components/popout_controller/popout_controller.scss

```diff
@@ -0,0 +1,7 @@
+body.popout {
+    background-color: var(--center-channel-bg);
+
+    .PopoutButton, .PopoutMenuItem {
```

Comment:
This is fine for now, but I'm not sure if it'll make long term sense to make the PopoutController responsible for changing the styling of elements in the app when the popout is open because it kind of has the same problem as the shared `_mobile.scss` where the CSS isn't with the component that's using it. We can always move that if we find it's an issue in the future